system/src/system_network_diagnostics.h

```diff
@@ -22,8 +22,23 @@
 
 namespace particle {
 
+// FIXME: Remove when alternate signal diagnostics are supported
+enum {
+  DIAG_TYPE_APP_INT = 1 << 11,
+  DIAG_TYPE_APP_UINT = 2 << 11,
+  DIAG_TYPE_APP_FIXED_S16_16 = 3 << 11,
+};
+
 class NetworkDiagnostics {
 public:
+    // FIXME: Share this enum on device service side?
+    enum NetworkInterface {
```

Comment:
Should this enum definition be in `system_defs.h` like the existing `network_interface_index` enum? 
And then use `NetworkInterface::NETWORK_INTERFACE_CELLULAR` etc instead of `NETWORK_INTERFACE_CELLULAR` ?